hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JetBootstrap.java

```diff
@@ -129,6 +130,15 @@ public static synchronized void executeJar(@Nonnull Supplier<JetInstance> suppli
         }
     }
 
+    private static Class<?> loadMainClass(ClassLoader classLoader, String mainClass) throws ClassNotFoundException {
+        try {
+            return classLoader.loadClass(mainClass);
+        } catch (ClassNotFoundException e) {
+            System.err.println("Cannot load main class: " + mainClass);
```

Comment:
should say cannot find or load main class, I think?

Reply:
Open question: what's the point of writing in the standard err if it's thrown anyway? Won't you get the unfound class name in the exception?

Reply:
we have a `verbose` option which is off by default and it was not very descriptive enough in that case